code/game/objects/effects/portals.dm

```diff
@@ -193,6 +193,28 @@
 	var/id // var edit or set id in map editor
 	hardlinked = FALSE // dont qdel my portal nerd
 
+/obj/effect/portal/permanent/teleport(atom/movable/M, force = FALSE) // Yogs start made perma portals always teleport even in noteleport areas since they are mapmaker only

```

Comment:
You could just move your copy of the proc over to the /yogstation/ folder, for modularity.